projects/plugins/jetpack/tools/webpack.config.extensions.js

```diff
@@ -26,19 +26,24 @@ const CopyBlockEditorAssetsPlugin = require( './copy-block-editor-assets' );
  */
 const editorSetup = path.join( path.dirname( __dirname ), 'extensions', 'editor' );
 const viewSetup = path.join( path.dirname( __dirname ), 'extensions', 'view' );
+const blockEditorDirectories = [ 'blocks', 'plugins' ];
 
 /**
- * Filters block scripts
+ * Filters block block scripts
  *
  * @param {string} type - script type
  * @param {string} inputDir - input directory
  * @param {Array} presetBlocks - preset blocks
  * @returns {Array} list of block scripts
  */
-function blockScripts( type, inputDir, presetBlocks ) {
-	return presetBlocks
-		.map( block => path.join( inputDir, 'blocks', block, `${ type }.js` ) )
-		.filter( fs.existsSync );
+function blockEditorScripts( type, inputDir, presetBlocks ) {
+	return _.uniq(
+		_.flatten(
+			blockEditorDirectories.map( dir =>
+				presetBlocks.map( block => path.join( inputDir, dir, block, `${ type }.js` ) )
+			)
+		)
```

Comment:
```suggestion
		blockEditorDirectories.flatMap( dir =>
			presetBlocks.map( block => path.join( inputDir, dir, block, `${ type }.js` ) )
		)
```

Reply:
Sorry, reverted. It doesn't work :-(

Reply:
sorry, testing again :-( Something is wrong with my local now.

Reply:
How didn't it work? I don't see any failing CI checks.